packages/core/src/RenderPipeline/SpriteRenderData.ts

```diff
@@ -1,36 +1,35 @@
-import { VertexData2D } from "../2d/data/VertexData2D";
+import { Primitive, SubMesh } from "../graphic";
 import { Material } from "../material/Material";
 import { Renderer } from "../Renderer";
 import { Texture2D } from "../texture";
-import { IPoolElement } from "./IPoolElement";
+import { IPoolElement } from "../utils/Pool";
+import { MBChunk } from "./batcher/MeshBuffer";
+import { RenderDataUsage } from "./enums/RenderDataUsage";
 import { RenderData } from "./RenderData";
 
 export class SpriteRenderData extends RenderData implements IPoolElement {
-  verticesData: VertexData2D;
   texture: Texture2D;
-  dataIndex: number; // Add for CanvasRenderer plugin.
+  chunk: MBChunk;
 
   constructor() {
     super();
-    this.multiRenderData = false;
+    this.usage = RenderDataUsage.Sprite;
   }
 
-  set(
+  override set(
```

Comment:
Further abstraction is needed. In addition, this is a common class between text and elves. The current definition is unreasonable.